src/plugin/manager/email/category_manager.py

```diff
@@ -37,7 +39,7 @@ def create_cloud_service(self, secret_data):
         for category in categories:
             reference = {
                     "resource_id": category.get("categoryId"),
-                    "external_link": "https://github.com/cloudforet-io/plugin-nhncloud-inven-collector"
+                    "external_link": f"https://cloudforet-dev.console.nhncloud.com/project/{secret_data.get('project_id')}/notification/email#manage-templates"
```

Comment:
For other organization, "cloudforet-dev.console.nhncloud.com" is not correct.

Reply:
This plugin is for some organizations who use nhncloud.

Reply:
@choonho I made mistake.
I found two issues.
When it comes to account with single user not organization, there is no additional sub domain for ex_link.
And to make ex_link for organization, we have to get additional organization name when register service account. I think it is inefficient only for ex_link.

Reply:
external_link is one of key information connecting cloudforet and real data. This kind of information is called as brokerage service. If there is no way to find external_link by API, exclude it.

Reply:
I update code.